### ISSUE TITLE
View the {Event} recording and presentation hyperlink

### ISSUE THREAD

**Issue by kevinkrzys** [theproductbase/theproductbaseclub.com#36]
## Current Observation
Currently, only the text "View" is a hyperlink that takes the user to the event page

## Desired Observation
The entire sentence should be a hyperlink to improve accessibility, both on desktop and mobile
